x-pack/legacy/plugins/reporting/index.js

```diff
@@ -182,22 +177,23 @@ export const reporting = (kibana) => {
       }).default();
     },
 
+    // TODO: Decouple Hapi: Build a server facade object based on the server to
+    // pass through to the libs. Do not pass server directly
```

Comment:
Added this comment to point out the next step to do for migration. See https://github.com/elastic/kibana/blob/master/src/core/MIGRATION.md#de-couple-from-hapijs-server-and-request-objects